.gitignore

```diff
@@ -25,3 +25,5 @@ hasjob/static/gen
 *.gz
 *.bz2
 hasjob/assets/node_modules
+admindash-config.cfg
+flask_monitoringdashboard.db
```

Comment:
We can't use SQLite in production as there are multiple worker processes competing for control over this file.

Reply:
By default it uses sqlite and creates this file. I moved it to postgres.